Controllers/TicketsController.cs

```diff
@@ -70,10 +76,13 @@ public async Task<IActionResult> CreateTicket(TicketForSendDto ticketForSendDto)
 
             return CreatedAtRoute("GetTicket", new { controller = "Tickets", id = createdTicket.Id }, ticketToReturn);
         }
-
+        
         [HttpGet]
         public async Task<IActionResult> GetTickets([FromQuery]TicketParams ticketParams)
         {
+            if(ticketParams.UserId != null)
+                ticketParams.UserId = _tokenService.GetTokenClaim(ticketParams.UserId, "nameid");
```

Comment:
Tu powinno być sprawdzone czy ticketParams.Token != null i jeśli tak to ogarnąć z tego UserId i wtedy przekazać to do metodki GetTickets, jako drugi parametr.